Error on dumps when running docker compose up

```bash
dumps              | 65000 out of 78394 50.094 sec
dumps              | INFO: A short_form (https://orcid.org/) starts with a digit: orcid_0000_0001_7258_9596 50.112 sec
dumps              | 66000 out of 78394 50.181 sec
dumps              | 67000 out of 78394 50.408 sec
dumps              | 68000 out of 78394 50.515 sec
dumps              | 69000 out of 78394 50.633 sec
dumps              | 70000 out of 78394 50.733 sec
dumps              | 71000 out of 78394 50.839 sec
dumps              | 72000 out of 78394 50.959 sec
dumps              | 73000 out of 78394 51.062 sec
dumps              | 74000 out of 78394 51.211 sec
dumps              | 75000 out of 78394 51.533 sec
dumps              | 76000 out of 78394 51.62 sec
dumps              | Exception in thread "main" java.lang.NumberFormatException: multiple points
dumps              |    at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1914)
dumps              |    at java.base/jdk.internal.math.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
dumps              |    at java.base/java.lang.Float.parseFloat(Float.java:455)
dumps              |    at uk.ac.manchester.cs.owl.owlapi.OWLLiteralImplNoCompression.parseFloat(OWLLiteralImplNoCompression.java:169)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OUtils.extractValueFromOWLAnnotationValue(N2OUtils.java:55)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OOntologyLoader.extractIndividualAnnotations(N2OOntologyLoader.java:157)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OOntologyLoader.extractSignature(N2OOntologyLoader.java:133)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OOntologyLoader.importOntology(N2OOntologyLoader.java:42)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OImportService.prepareCSVFilesForImport(N2OImportService.java:35)
dumps              |    at ebi.spot.neo4j2owl.importer.N2OImporterRunner.main(N2OImporterRunner.java:24)
dumps              | make: *** [Makefile:98: pdb_csvs] Error 1
dumps exited with code 2
Gracefully stopping... (press Ctrl+C again to force)
service "obask-dumps" didn't complete successfully: exit 2
```

What could that be?

--- a/config/collectdata/vfb_fullontologies.txt
+++ b/config/collectdata/vfb_fullontologies.txt
@@ -1,2 +1,2 @@
-http://purl.obolibrary.org/obo/uberon/composite-metazoan-basic.owl
-https://raw.githubusercontent.com/EBISPOT/scxa_2_cxg/main/efo_slice.owl
+http://purl.obolibrary.org/obo/uberon/composite-metazoan.owl
+https://github.com/EBISPOT/scxa_2_cxg/raw/main/efo_slice.owl

